.github/workflows/codeball.yml

```diff
@@ -0,0 +1,21 @@
+name: Codeball
+on:
+  pull_request: {}
+  pull_request_review_comment:
+    types: [created, edited]
+
+jobs:
+  codeball_job:
+    runs-on: ubuntu-latest
+    name: Codeball
+    steps:
+      # Run Codeball on all new Pull Requests and Review Comments! 🚀
+      # For customizations and more documentation, see https://github.com/sturdy-dev/codeball-action
+      - name: Codeball
+        uses: sturdy-dev/codeball-action@v2
+        with:
+          approvePullRequests: "true"
+          labelPullRequestsWhenApproved: "true"
+          labelPullRequestsWhenReviewNeeded: "false"
+          failJobsWhenReviewNeeded: "false"
+          codeSuggestionsFromComments: "true"          
```

Comment:
Not generate the comment? How to trigger the suggestion action?
codeSuggestionsFromComments